src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ExternalContracts.cs

```diff
@@ -9,6 +9,8 @@
 using NuGet.SolutionRestoreManager;
 using NuGet.VisualStudio;
 
+#pragma warning disable RS0030 // Do not used banned APIs
```

Comment:
Which banned API was it complaining about?

Reply:
Three different ones. I didn't think it was useful to highlight them in this file.